### ISSUE TITLE
Styling and responsiveness needs a couple passes

### ISSUE THREAD

**Issue by jlhall** [jlhall/D3Rand_Builder#16]
All classes will theoretically be buildable in the same session, need a better way to display them than flex/space around in CSS...possibly keep flex but reorient Button component display (alter img display type?)

**Comment by jlhall** [comment]
Styling somewhat fixed with #31 

Leaving open until minimum browser window size functions well

Especially skill images etc, these need to get smaller at different viewport sizes etc.

~ * CoMpLeTe rEsPoNsIvEnEsS WaNtEd* ~

**Comment by jlhall** [comment]
Some more styling done with 71ae17f, but introduced a ugly 🐛 , delaying release of made-over app to production. Tooltips will be brought up as a separate issue after #28 and #29 are closed, as that is an issue styling can't necessarily fix. I'm currently thinking about a touch and hold tooltip script for mobile devices, and leaving the regular mouse over functionality for standard browsers.

**Comment by jlhall** [comment]
Working responsively with 83d1a65, will open tooltip concerns in new issue
